app/src/main/java/lt/vilnius/tvarkau/AboutActivity.java

```diff
@@ -126,4 +127,27 @@ private void openEmail(String emailAddress) {
                 .show();
         }
     }
+
+    private String getDeviceInfo() {
+        String manufacturer = Build.MANUFACTURER;
+        String model = Build.MODEL;
+        String androidVersion = Build.VERSION.RELEASE;
+        if (model.startsWith(manufacturer)) {
+            return " (" + capitalize(model) + ", " + "Android " + androidVersion + ")";
+        } else {
+            return " (" + capitalize(manufacturer) + " " + model + ", " + "Android " + androidVersion + ")";
+        }
+    }
+
+    private String capitalize(String s) {
```

Comment:
In my opinion, `getDeviceInfo()` `capitalize()` methods are util methods and semantically has nothing to do with Activity. Thus, I would suggest to move these to one of the utility class.